src/ArtifactUtils.jl

```diff
@@ -73,7 +75,7 @@ function add_artifact!(
         artifacts_toml,
         name,
         git_tree_sha1;
-        download_info = [(tarball_url, sha256)],
+        download_info = [(tarball_url, sha256, tarball_size)],
```

Comment:
I was a little surprised to see this works, I suppose it's good I didn't specify the types in the tuple originally!